File: app/src/main/java/com/example/authapptutorial/main_navigation/ApiService.java
package com.example.authapptutorial.main_navigation;

import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;
import retrofit2.Call;

public interface ApiService {

    @FormUrlEncoded
    @POST("chat")
    public void chatWithTheBit(
            @Field("chatInput")
                    String chatText);


public void ChatReponse(String chatbotReply);

}

File: app/src/main/java/com/example/authapptutorial/main_navigation/ApiService.kt
package com.example.authapptutorial.main_navigation

import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

interface ApiService {

    @FormUrlEncoded
    @POST("/chat")
    fun chatWithTheBit(@Field("chatInput") chatText : String ): Call<ChatResponse>
}
data class ChatResponse(val chatBotReply: String)